packages/version-bridge/src/lib/VersionedValue.ts

```diff
@@ -36,9 +36,27 @@ export function createVersionedValueMap<
   Versions extends { [version: number]: unknown },
 >(versions: Versions): VersionedValue<Versions> {
   Object.freeze(versions);
-  return {
+  const versionedValue: VersionedValue<Versions> = {
     atVersion(version) {
       return versions[version];
     },
   };
+  Object.defineProperty(versionedValue, '$value', {
+    configurable: false,
+    enumerable: true,
```

Comment:
I would've actually expected this to be the other way around? Feels a bit mean to disable configuration, and we don't really need them to be enumerable for this use case? I like the idea of them being a bit more hidden.

Reply:
I added configurable, but if it's not enumerable, it won't show up in dev tools (or console log, toJson, etc).